worlds/sc2/__init__.py

```diff
@@ -458,21 +441,6 @@ def place_exclusion_item(item_name, location, locked_locations, player):
     locked_locations.append(location.name)
 
 
-def get_exclusion_item(multiworld: MultiWorld, option) -> str:
-    """
-    Gets the exclusion item according to settings (trash/nothing)
-    :param multiworld:
-    :param option:
-    :return: Item used for location exclusion
-    """
-    if option == LocationInclusion.option_nothing:
-        return "Nothing"
-    elif option == LocationInclusion.option_trash:
-        item = multiworld.random.choice(filler_items)
-        return item
-    raise Exception(f"Unsupported option type: {option}")
-
-
 def get_plando_locations(multiworld: MultiWorld, player) -> List[str]:
```

Comment:
Drop that if you moved that function to Locations.py

Reply:
Done.